plasma_framework/migrations/2_deploy_plasma_framework.js

```diff
@@ -2,16 +2,21 @@ const PlasmaFramework = artifacts.require('PlasmaFramework');
 
 const config = require('./config.js');
 
-module.exports = async (deployer) => {
+module.exports = async (
+    deployer,
+    _,
+    // eslint-disable-next-line no-unused-vars
```

Comment:
can we use `_` if the variable is not used to get pass through the linter instead?

Reply:
Unfortunately the linter doesn't support multiple `_` :|